tests/main/presenters/test_search_presenters.py

```diff
@@ -256,7 +256,7 @@ def test_show_lots_and_categories_selection(self, search_api_client):
         with self.app.test_request_context(url):
```

Comment:
This long test feels like it could be split in two (with maybe some shared setup/context)?